exec/java-exec/src/test/java/org/apache/drill/test/rowSet/RowSet.java

```diff
@@ -85,8 +85,7 @@
    * new row set with the updated columns, then merge the new
    * and old row sets to create a new immutable row set.
    */
-
-  public interface RowSetWriter extends TupleWriter {
+  interface RowSetWriter extends TupleWriter {
```

Comment:
Aren't nested interfaces `protected` by default? Just had to change one from default to `public` so I could use it in another package...

Reply:
IntelliJ gave a warning that the modifier is redundant. Also an interface nested inside another interface is public by default.

https://beginnersbook.com/2016/03/nested-or-inner-interfaces-in-java/

Reply:
Ah, forgot that the file defines an interface, not a class. (The situation I described was an interface nested inside a class.) So, you're good.